package.json

```diff
@@ -13,7 +13,9 @@
     "styleguide": "cd ./styleguide && yarn dev",
     "styleguide:build": "cd ./styleguide && yarn build:lib && cd ../",
     "test": "jest",
-    "precommit": "yarn lint"
+    "precommit": "yarn lint",
+    "e2e:local": "cypress run --headed",
+    "e2e:ci": "npm-run-all --parallel --race start:ci 'cypress:ci --config baseUrl=http://localhost:3000'"
```

Comment:
Cypress is installed globally, if you want to keep it that way, please add `cypress` to the development dependencies and change `.travis.yml` to use it on our build server.